docs/modules/ROOT/pages/release-notes/release-notes-21-2.adoc

```diff
@@ -0,0 +1,28 @@
+== Supported APIs and Applications
+
+* Java EE 8 Applications
+* MicroProfile 3.3
+* Jakarta EE 8 Applications
+
+*Notes*
+
+In this release of Payara Enterprise we're unfortunately temporarily reverting an improvement we added in 5.21 - HTTP/2 support for JDK Native ALPN APIs (FISH-31). It has proved to be unstable in certain situations and so we've taken the decision to remove it from the Enterprise distributions until it's matured further - we were too quick to bring this feature in to Enterprise and apologise if this impacts you. This reversion means that HTTP/2 support will not work when using JDK 8u252 or higher (including JDK 11), and a warning will be printed in the log related to this. HTTP/2 should still work when using an older supported version of JDK 8 (8u162-8u242).
+
+*Bug Fixes*
+
+* [https://github.com/payara/Payara-Enterprise/pull/171[_FISH-37_]] - @DataSourceDefinition passes serverName and URL DataSource in some cases
```

Comment:
Other release notes don't have Jira number in Italic. Can be fixed in another PR.